lib/index.js

```diff
@@ -0,0 +1,12 @@
+const _getAbstraction = (name) => {
```

Comment:
We should always try to use strict mode when possible. Suggest adding 'use strict'; statement to all .js files.

Reply:
Fixed